pkg/tsdb/cloudwatch/log_actions_test.go

```diff
@@ -406,33 +406,6 @@ func Test_executeStartQuery(t *testing.T) {
 		}, cli.calls.startQueryWithContext)
 	})
 
-	t.Run("cannot parse limit as float", func(t *testing.T) {
```

Comment:
I'm a little worried about why we had this test, because if getting a float limit is possible, we're going to error unmarshaling it.

Reply:
Yeah, same.

Reply:
I think it was a test as documentation of some behavior or consequence of the behavior of the simplejson library but not sure. I vote for deletion!